styles/styles.css

```diff
@@ -306,6 +306,7 @@ header {
 }
 
 body {
+  display: none;
```

Comment:
@ravverma  can you please explain this change a little bit ?

Reply:
First I have tried the sunstar's project and find out that in their project which had older boiler plate code and in the `libs-franklin.js#init()` , the body display style was setting to `none` and after the `waitForLCP` it was set to `null` , so you won't see anything before decorate.
In our code this line `document.body.style.display = 'none';` is missing which was part of the init() method in renamed `aem.js` but it was removed by this https://github.com/adobe/aem-boilerplate/pull/224/files saying duplicate but i didn't find it duplicate so I will try to reach to contributor as well.
then I have added it again which shows improvements but compare to sunstar we have lots of text on initial viewport so the text was still showing because before init() it loads the plain DOM.
so for the experiment POV we set default body style to `display:none` and it is removed in `waitForLCP` which is already part of our `aem.js`
but it is degrading the performance from 100 -> 93.
So I am  still on the issue and trying optimal solution. 
cc: @shrotia @sdmcraft

Reply:
fair enough @ravverma  , I have updated the label as draft on this pr

Reply:
@ravverma , I think the direction is right though I do not remember the specifics. The idea is to keep everything hidden and as the blocks get loaded, start showing them up. The block's `loaded` status should make them visible through CSS styling.

Reply:
We had originally `display:none` set on body when we started the project. [See here](https://github.com/aemsites/icicidirect/commit/87a2fe0b9bb54f1b747c0af5b9fa73e3e4dd3895#diff-37f6471b306924bbb531d040e7ab55df5f83511246a411d22947462323516afaR69). But I don't find it any longer now, so it got removed via some commit. Could you check?

Reply:
I think this commit removed it in past
https://github.com/aemsites/icicidirect/commit/1fde3174aea3ec272dee2c87f7e2e86532afea04#diff-37f6471b306924bbb531d040e7ab55df5f83511246a411d22947462323516afa